build.py

```diff
@@ -80,6 +80,11 @@ def build():
                              utility.ROOT_DIR,
                              continue_on_error=False)
 
+        # generate sdist--only run on Mac to eliminate redundant copies when published to Azure
+        if sys.platform == 'darwin':
```

Comment:
please document that mac wasn't chosen for a reason (you just needed to choose one platform) to avoid future confusion